### ISSUE TITLE
Accessibility check

### ISSUE THREAD

**Issue by Faithchidindu** [oppia/oppia-android#4941]
### Is your feature request related to a problem? Please describe.

1. Learners struggle to read the "stories you can play" text because it has a white text on as grey background and thus did not pass the WCAG accessibility check. This poses as a problem for user with eye defect. Some of the buttons have their text in black and with a dark background.

2. There is inconsistency in the UI theme. Some of the screens are in light mode while some are in dark mode. There is no uniformity

### Describe the solution you'd like

It would be better if the text was given a black colour.
White text should be used on a dark background, dark text should be used on white background inorder to make the text easy to read

The user interface will look much better if the user is allowed to switch between light and dark themes. This creates a sense of uniformity, reliability and control

### Additional context

![Screenshot_20230331-174037](https://user-images.githubusercontent.com/127211589/230753981-4efbeb04-6e5e-4439-83d3-8e0e7b85a68c.jpg)
![Screenshot_20230331-174253](https://user-images.githubusercontent.com/127211589/230754004-e67ba31c-1732-4e5f-abf7-654aae963429.jpg)

**Comment by prafulbbandre** [comment]
@Faithchidindu Please open this issue in oppia-Android repository.

**Comment by seanlip** [comment]
Transferred to Android repo -- thanks @prafulbbandre.

**Comment by MohitGupta121** [comment]
@Faithchidindu Thanks for this issue. I think you are using the app from Playstore.
But this issue is fix in this PR https://github.com/oppia/oppia-android/pull/4796 and soon it will Beta release in Google Playstore. 
You can get the build from latest develop branch (if possible or I will share debug apk ?) 

Here is latest screenshots of Accessibility check from develop branch:

<img src="https://user-images.githubusercontent.com/76530270/230964286-27f56bc2-76c9-4847-8e35-9bbe57270773.png" height="400" style="max-width: 100%">  <img src="https://user-images.githubusercontent.com/76530270/230964270-874fdbb8-f92e-4b71-84ef-e3eaa8877b53.png" height="400" style="max-width: 100%"> 


Thanks @seanlip @prafulbbandre for transferring this issue.

**Comment by MohitGupta121** [comment]
@seanlip we can close this issue ?

**Comment by Faithchidindu** [comment]
Thanks for the clarification @MohitGupta121

**Comment by BenHenning** [comment]
I'm fine with moving this to the 0.11 release list so that it can be verified as fixed. Thanks again for reporting @Faithchidindu and thanks @MohitGupta121 for checking that this has been fixed!

**Comment by KolliAnitha** [comment]
Working fine in 0.11-beta-8e0fcf19ac
![name ](https://github.com/oppia/oppia-android/assets/101634267/c5937b56-dc49-49c4-b4ac-b689a74b5d86)
